routers/web/passkey.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package web
+
+import (
+	"net/http"
+	"path"
+
+	"code.gitea.io/gitea/modules/context"
+	"code.gitea.io/gitea/modules/setting"
+)
+
+type passkeyEndpoints struct {
+	Enroll string `json:"enroll"`
+	Manage string `json:"manage"`
+}
+
+func PasskeyEndpoints(ctx *context.Context) {
+	ctx.JSON(http.StatusOK, passkeyEndpoints{
+		Enroll: path.Join(setting.AppURL, setting.AppSubURL, "user/settings/security"),
+		Manage: path.Join(setting.AppURL, setting.AppSubURL, "user/settings/security"),
+	})
```

Comment:
```suggestion
	url := setting.AppURL + "user/settings/security"
	ctx.JSON(http.StatusOK, passkeyEndpoints{
		Enroll: url,
		Manage: url,
	})
```

Reply:
On a related note:
`setting.AppURL` or `setting.AppSubURL`?
I haven't found a definitive answer if relative URLs are allowed or not, and given there are some people running Gitea across multiple instances, `AppURL` would not work for them…

Reply:
Well the example indicates absolute url